lib/etc/openturns.conf.in

```diff
@@ -458,15 +458,17 @@
 
   <!-- OT::GeneralizedLinearModelAlgorithm parameters -->
   <GeneralizedLinearModelAlgorithm-LinearAlgebra                  value="LAPACK"  />
-  <GeneralizedLinearModelAlgorithm-NormalizeData                  value="false" />
-  <GeneralizedLinearModelAlgorithm-KeepCovariance                 value="true" />
-  <GeneralizedLinearModelAlgorithm-OptimizeParameters             value="true" />
+  <GeneralizedLinearModelAlgorithm-NormalizeData                  value="false"   />
+  <GeneralizedLinearModelAlgorithm-KeepCovariance                 value="true"    />
```

Comment:
I don't agree with your proposal. The GLMAlgo is not always used through the KrigingAlgorithm class, in which case one could décidé not to keep the huge objects.